lib/Models/REST/RestClient.php

```diff
@@ -4,10 +4,12 @@
  */
 namespace Opencast\Models\REST;
 
+use GuzzleHttp\HandlerStack;
 use Opencast\Models\Config;
 use Opencast\Errors\RESTError;
 use OpencastApi\Opencast;
 use OpencastApi\Rest\OcRestClient;
+use Opencast\Middlewares\REST\ConnectionMiddlewares;
 
 use \Context;
 use \Config as StudipConfig;
```

Comment:
Better sort all `use`s.
```suggestion
use Config as StudipConfig;
use Context;
use GuzzleHttp\HandlerStack;
use OpencastApi\Opencast;
use OpencastApi\Rest\OcRestClient;
use Opencast\Errors\RESTError;
use Opencast\Middlewares\REST\ConnectionMiddlewares;
use Opencast\Models\Config;
```